src/custom.d.ts

```diff
@@ -16,9 +16,13 @@
 
 export {}
 
-declare global{
+declare global {
   namespace Express {
     export interface Request {
+      params: {
```

Comment:
Is this needed?

Reply:
The base express had `string` as the key type, so it wouldn't let me refer to `0`... that was at least what my local setup was complaining about.